ansible/roles/go-cli/tasks/deploy.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Tasks for handling Go CLI customization and publishing
+
+- name: "Ensure go-cli directory exists"
+  file:
+    path: "{{ go_cli_conf_dir }}"
+    state: directory
+
+- name: "Pull the {{ docker_image_tag }} image of go-cli"
+  shell: "docker pull {{ docker_registry }}whisk/go-cli:{{ docker_image_tag }}"
+  when: docker_registry != ""
+
+# TODO: Publish compressed binaries using nginx
```

Comment:
@lehoanganh, we have @houshengbo working on the publishing aspect mentioned here.